CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.13
+* Tests now ensure that composer.lock is kept up to date. (GitHub #132)
+* The page title block is now properly placed in the Seven theme. (GitHub #190)
+* Twitter media entities now have automatically generated thumbnails.
+  (GitHub #203)
+* CI now uses database snapshots for update tests. (GitHub #201)
+* Fix multiversion regression of node revisions list. (Issue #2824633)
+* Display modes and user roles can now have associated descriptions.
+  (GitHub #195)
+* Improved usability of the panelizer interface. (Issue #2826071)
+* Added debug steps and stabilized finicky test. (GitHub #218)
```

Comment:
Not sure we need to mention this? It was, after all, strictly an internal change.